js/index.js

```diff
@@ -0,0 +1,72 @@
+import books from "./data.js"; // eslint-disable-line
+
+const myLibrary = [];
+
+function Book(title, author, pages, status = false) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.status = status;
+}
+
+const addBookToLibrary = (title, author, pages) => {
+  const book = new Book(title, author, pages);
+  myLibrary.push(book);
+  renderTable(); // eslint-disable-line
+};
```

Comment:
## Add `read` option to the form submission.
- [ ] During book creation, kindly add the option to add a book. The form should allow the marking the `Book` status as `read`